android/src/google/play/listings/en-US/full-description.txt

```diff
@@ -0,0 +1,38 @@
+• Detailed offline maps with places that don't exist on other maps
+• Cycling routes, hiking trails and walking paths
+• Contour lines, elevation profiles, peaks and slopes
+• Turn-by-turn walking, cycling and car navigation with voice guidance
+• Fast offline search on the map and bookmarks
+• Dark mode to protect your eyes
```

Comment:
• Detailed offline maps with places that don't exist on other maps, thanks to OpenStreetMap
• Cycling routes, hiking trails, and walking paths
• Contour lines, elevation profiles, peaks, and slopes
• Turn-by-turn walking, cycling, and car navigation with voice guidance
• Fast offline search on the map
• Bookmarks export and import in KML/KMZ formats (GPX is planned)
• Dark Mode to protect your eyes
• Countries and regions don't take a lot of space
• Free and open-source